core-bundle/contao/themes/flexible/styles/layout/filter.pcss

```diff
@@ -119,8 +120,8 @@
     display: flex;
     flex-wrap: nowrap;
     gap: 10px;
-    padding: 10px 15px;
-    border-radius: 0 0 var(--border-radius) var(--border-radius);
+    padding: 10px var(--gap);
```

Comment:
I don't see how `18px` is better than `15px`. It makes almost no difference on big screens and takes away precious space on small screens.